crates/state/src/style.rs

```diff
@@ -132,6 +150,7 @@ impl State<CustomAttributeValues> for StyleState {
     const NODE_MASK: NodeMaskBuilder<'static> =
         NodeMaskBuilder::new().with_attrs(AttributeMaskBuilder::Some(&[
             AttributeName::Background,
+            AttributeName::Fill,
```

Comment:
You are missing Stroke right?

```suggestion
            AttributeName::Fill,
            AttributeName::Stroke,
```